site/assets/scss/_component-examples.scss

```
@@ -401,7 +400,7 @@
 
 .bd-scss-docs {
   .highlight-toolbar {
-    @include border-top-radius(calc(var(--bs-border-radius) + 1px));
+    @include border-top-radius(calc(var(--bs-border-radius) - 1px));
```

```suggestion
    @include border-top-radius(var(--bs-border-radius));
```

Looks better to me with the `border-radius` increased in `.bd-code-snippet` 🤷